docs/run/hardware.mdx

```diff
@@ -1,101 +1,100 @@
 ---
 title: System information
-description: Information about IBM Quantum hardware, features of different processors, and system calibration, properties, and versioning
+description: Information about IBM Quantum system calibration, properties, and versioning
 
 ---
 
 # System information
 
-IBM Quantum offers both open and premium access to a wide variety of quantum systems. All quantum systems deployed by IBM Quantum are based on superconducting qubit technology, as the control and scalability of this technology pave a clear path to achieving quantum advantage with these systems. You can see the full details of all IBM Quantum systems on the [Compute resources page](https://quantum-computing.ibm.com/services/resources?tab=systems).
+IBM Quantum offers both open and premium access to a wide variety of quantum systems. All quantum systems deployed by IBM Quantum are based on superconducting qubit technology, as the control and scalability of this technology pave a clear path to achieving quantum advantage with these systems. You can see the full details of all IBM Quantum systems on the [Compute resources page.](https://quantum-computing.ibm.com/services/resources?tab=systems)
```

Comment:
Why this change? Punctuation should be in the hyperlink?

Reply:
When the period comes after the hyperlink, it puts an extra space there, and can also mean a linebreak between the link and the period depending on the line length.